README.md

```diff
@@ -1,2 +1,10 @@
-# ruby-homework
+# readme for ruby-homework
 A repository I will use to solve my Ruby homework
+I am following Ruby course [Learn Ruby](https://github.com/monorkin/learn.rb)
+## My favorite programming languages
+* __Python__
+* __Java__
+* __C__
+  * __C++__
```

Comment:
Was this indent intentional? This way you created a 2nd level in the list.
Either way, task well done.

Reply:
Yup it was intentional, that way it marks that c++ is sort of upgrade on c.